docs: change chain to table (#415)
oasisprotocol/sapphire-paratime: docs/network.mdx

---

> description: Detailed information for Sapphire about chains, RPCs, block explorer and indexer

Reviewer: ```suggestion
description: Detailed information for Sapphire about chains, RPCs, block explorers and indexers
```

---

import {AddSapphireToMetaMask as S, AddSapphireTestnetToMetaMask as ST} from '@site/src/AddToMetaMask';

# Network information

## Networks

|                   | Mainnet                             | Testnet                             | Localnet                                   |
| ------------------| ----------------------------------- | ----------------------------------- | ------------------------------------------ |
| Network name      | `sapphire`                          | `sapphire-testnet`                  | `sapphire-localnet`                        |

> | Long network name | `Oasis Sapphire`                    | `Oasis Sapphire Testnet`            | `Oasis Sapphire Localnet`                                 |

Reviewer: ```suggestion
| Long network name | `Oasis Sapphire`                    | `Oasis Sapphire Testnet`            | `Oasis Sapphire Localnet`                  |
```

id: network
title: Network Information
description: Detailed information about chains, RPCs, block explorer and indexer

> sidebar_label: Network Information

Reviewer: Let's only keep the `description` field here. The sidebar label, title and id should be set in the sidebar.js.

---

import {AddSapphireToMetaMask as S, AddSapphireTestnetToMetaMask as ST} from '@site/src/AddToMetaMask';

# Network information

## Networks

|                   | Mainnet                             | Testnet                             | Localnet                                   |
| ------------------| ----------------------------------- | ----------------------------------- | ------------------------------------------ |
| Network name      | `sapphire`                          | `sapphire-testnet`                  | `sapphire-localnet`                        |

> | Long network name | `Oasis Sapphire`                    | `Oasis Sapphire Testnet`            | `Localnet`                                 |

Reviewer: Is there a reason for not calling the local chain `Oasis Sapphire Localnet`?

```suggestion
| Long network name | `Oasis Sapphire`                    | `Oasis Sapphire Testnet`            | `Oasis Sapphire Localnet`                                 |
```

Reviewer 2: I do like "Sapphire Localnet" which we have in the web3 gateway [README](https://github.com/oasisprotocol/oasis-web3-gateway/blob/c41077a8cf8ef93d6d16fe6124e38252c058d86a/docker/README.md?plain=1#L58).
